string[] layers
geographic_msgs/GeoPose[] fov
---
enc_query/enc_feature_msg[] featuresInView